net/proxy_resolution/proxy_config_service_tor.cc

```diff
@@ -137,13 +137,13 @@ std::string ProxyConfigServiceTor::CircuitIsolationKey(const GURL& url) {
   //
   // In particular, we need not isolate by the scheme,
   // username/password, port, path, or query part of the URL.
-  url::Origin origin = url::Origin::Create(url);
-  std::string domain = net::registry_controlled_domains::GetDomainAndRegistry(
-      origin.host(),
-      net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES);
-  if (domain.size() == 0)
-    domain = origin.host();
-  return domain;
+  url::Origin url_origin = url::Origin::Create(url);
+  net::NetworkIsolationKey network_isolation_key(url_origin, url_origin);
+
+  const base::Optional<url::Origin>& network_isolation_key_origin =
+    network_isolation_key.GetEffectiveTopFrameOrigin();
```

Comment:
lint nit: 4 spaces for continuation of a line

Reply:
Oops! Sorry, will change that